mfa/oath/src/androidTest/kotlin/com/pingidentity/mfa/oath/OathClientAndroidTest.kt

```diff
@@ -96,11 +95,11 @@ class OathClientAndroidTest {
                 client.initialize()
```

Comment:
Do we need to initialize()? The OathClient already initialize it?
Similar issue apply to other test

Reply:
Fixed